### ISSUE TITLE
[Bug] plq_Ridge_Regressor raises Exception for 'check_eps' loss

### ISSUE THREAD

**Issue by BenViolet6** [softmin/ReHLine-python#36]
## Bug Description
I am trying to use plq_Ridge_Regressor with the 'check_eps' (epsilon-insensitive check loss) loss function. However, the code raises an exception in _make_loss_rehline_param, stating that the loss function is not supported.

It seems the string matching for the loss name 'check_eps' might be failing, or the wrapper is missing the branch to handle this specific loss type, even though it appears to be a valid configuration for the underlying solver.

## Expected Behavior
The model should successfully initialize and fit using the 'check_eps' (Quantile Regression with Epsilon Tolerance) loss function, as this loss type seems intended to be supported.

## Actual Behavior
The code raises an Exception during clf.fit(X=X, y=y). The traceback shows the error occurs inside _make_loss_rehline_param, with the message: Exception: Sorry, ReHLine currently does not support this loss function...

## Steps to Reproduce
1. Generate synthetic regression data with noise.
2. Initialize plq_Ridge_Regressor with loss={'name': 'check_eps', 'qt': 0.9, 'epsilon': 0.1}.
3. Call clf.fit(X, y)

## Minimal Code Example

```python
# Minimal reproducible example
import numpy as np
from rehline import ReHLine

import numpy as np
from rehline import plq_Ridge_Regressor

# 1. Simulate data 
np.random.seed(42)
n = 2000
x = np.random.randn(n)
noise = np.random.randn(n) * (0.3 + 0.5 * np.abs(x))
y = 2 * x + noise
X = x.reshape(-1, 1)

# 2. Check Loss with epsilon-tolerance
# This line causes the crash
clf = plq_Ridge_Regressor(loss={'name': 'check_eps', 'qt': 0.9, 'epsilon': 0.1}, C=10.0/n)
clf.fit(X=X, y=y)
```

## Environment

- OS: Linux / 6.6.113+
- Python version: 3.12.12
- ReHLine version: Not found (or check pip show rehline)
- NumPy version: 2.0.2
- SciPy version: 1.16.3
- scikit-learn version: 1.6.1
